tests/hardware_reboot/testdata/hardware_usb_reboot.txt

```diff
@@ -13,20 +11,23 @@ exec -t 20m bash ssh.sh 2223
 stdout 'Ubuntu'
 
 exec -t 20m bash get-lshw.sh 2223
-cp stdout before_reboot
+stdout '1024B QEMU HARDDISK'
 
-exec -t 20m bash get-usb.sh 2223
-grep 'QEMU USB HARDDRIVE' {{$usb_dev}}.usb.product
+exec -t 20m bash get-lshw.sh 2223
+cp stdout before_reboot
 
 # reboot from guest
 exec -t 20m bash reboot.sh 2223
-exec sleep 10
+exec sleep 60
+
+# reboot detector
+test eden.reboot.test -test.v -timewait 20m -reboot=1
```

Comment:
It is reboot detector for EVE. And in such notation it will wait for reboot for 20 minutes. Is it expected behavior here?

Reply:
Does it seem reasonable to check the USB transfer status in both cases - after stopping/starting the application and also after a full EVE restart?

Reply:
The issue we've had in the past was that the handling of a reboot from inside the guest failed to re-create the device passthrough. That is a completely different code path at least in Xen than the normal stop/start of the guest.

Rebooting the device and starting the application/guest a second later isn't any different than booting the device and starting the application/guest ten minutes later. So testing that isn't critical.

Reply:
OK, @eriknordmark  just a final clarification -- we need a test that passes on EVE-xen with checking the USB-passthrough status after rebooting the application. It is right?